R/options.R

```diff
@@ -123,3 +123,70 @@ sass_options <- function(
   class(ret) <- c("sass_options", class(ret))
   ret
 }
+
+#' Caching Options for Sass
+#'
+#' Specifies whether caching is used with sass, and where on the file system the
+#' cached data should be stored. Used with \code{\link{sass}}.
+#'
+#' If caching is enabled, \code{sass()} will attempt to bypass the compilation
+#' process by reusing output from previous \code{sass()} calls that used
+#' equivalent inputs. This mechanism works by using a hashing algorithm to
+#' derive a \emph{cache key} from each \code{sass()} call's \code{input} and
+#' \code{options} arguments. If a file named
+#' \emph{CACHE_KEY}\code{.sasscache.css} exists within the cache directory, its
+#' contents are used instead of performing the compilation. If the file does not
+#' exist, then compilation is performed and usual and the results are stored at
+#' that file path for next time.
```

Comment:
Add a note about detection of changes in files (the fact that it detects changes in `sass_file` files, but not in files imported by those files).